redbox-core/redbox/loader/loaders.py

```diff
@@ -135,77 +62,30 @@ def _chunking(self) -> Any:
         if response.status_code != 200:
             raise ValueError(response.text)
 
-        return response.json()
+        return response.json() or []
 
-    def extract_metadata(self):
+    def extract_metadata(self) -> dict:
         """
         Extract metadata from first 1_000 chunks
         """
 
-        elements = self._chunking()
-        if not elements:
-            self.metadata = self.default_metadata
-
-        else:
-            # Get first 1k tokens of processed document
-            first_n = self.get_first_n_tokens(elements, 1_000)
+        chunks = self._chunking()
+        first_thousand_words = "".join(chunk["text"] for chunk in chunks)[:1_000]
```

Comment:
This is the first 1000 characters not tokens, I think we need to retain the get_first_n_tokens function to do that?

Reply:
what if it was 10,000 chars? either way its just a made up number